### PR TITLE
Wadenswil map

### PR DESCRIPTION
**Pull request by jop22**
@napszel 
Please have a look when you have time on jquery approach to sectors highlighting. If you like it I will adjust polygons and on-hover styling.
First commit in this branch fixes passing sector name to the table. I broke it in my previous branch when I changed sector name column being displayed from the old name to VLSector. Sorry. :(
Old maps need some fixes to handle new sector names. I will fix it later.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -116,14 +116,14 @@ table.dataTable tbody td {
...
     margin-left: 15px;
     color: white;
-    padding: 6px 20px;
+    padding: 6px 10px;
     text-align: center;
     text-shadow: 2px 2px 5px black;
-    width: 23.2%;
-    height: 100%;
+    width: 15%;
+    height: 85%;
```
Comment (napszel) [inline style.css:126]: Sorry, I changed this to 85%. This makes the button text to be in the middle. At least on my screen. :)

**Comment by napszel** [comment]
Thanks so much! This is awesome! I definitely want to merge this right now as it's so useful already. I can write comments on routes now. :)

Do you know if it would be possible to make the maps resize on smaller screens? On my laptop they look like this (browser in full screen; note the scroll bars):
Screenshot 2021-07-11 at 00 34 27
Screenshot 2021-07-11 at 00 34 09
Not a deal breaker, just asking. On my screen it would only fit if it were not wider than around 1200px.

File: style.css (hunk 1 above)
Comment (jop22) [reply style.css:126]: That's better. On my screen as well. I'm css newbie.

**Comment by jop22** [comment]
I will add missing tasks in Trello so I don't forget them. Including this resizing.. No idea if it is possible, Should be ...

**Pull request merged**